arcgis-android-toolkit/build.gradle

```diff
@@ -81,8 +81,10 @@ dependencies {
     implementation "androidx.appcompat:appcompat:$droid.appcompat"
     implementation "androidx.constraintlayout:constraintlayout:$droid.constraintLayout"
     implementation "androidx.multidex:multidex:$droid.multiDex"
+    implementation 'androidx.navigation:navigation-fragment-ktx:2.3.3'
     implementation "androidx.lifecycle:lifecycle-runtime:$droid.lifecycle"
     implementation "androidx.recyclerview:recyclerview:$droid.recyclerview"
+    implementation 'com.google.android.material:material:1.3.0'
```

Comment:
define the version in https://github.com/Esri/arcgis-runtime-toolkit-android/blob/master/gradle/script/versions.gradle

Reply:
updated